src/pages/Actions/ActionEvent/EventInfo/EventInfo.tsx

```diff
@@ -174,32 +199,34 @@ const EventInfo = () => {
           <Gallery
             key={event.event?.eventLocation}
             eventId={event.event?.eventId}
-            isUserEventAdmin={event.isUserEventAdmin}
+            userAccesses={userAccesses}
           />
         </div>
-        <div className="participantsTable">
-          <div key={"2"}>
-            <Title level={2} style={{ color: "#3c5438" }}>
-              Таблиця користувачів
-            </Title>
-            <Row>
-              <Input.Search
-                style={{ width: "400px", margin: "0 0 10px 0" }}
-                placeholder="Пошук"
-                enterButton
-                onSearch={search}
+        {userAccesses["SeeUserTable"] || event.isUserApprovedParticipant ?
+          <div className="participantsTable">
+            <div key={"2"}>
+              <Title level={2} style={{ color: "#3c5438" }}>
+                Таблиця користувачів
+              </Title>
+              <Row>
+                <Input.Search
+                  style={{ width: "400px", margin: "0 0 10px 0" }}
```

Comment:
It's highly recommended to write styles in .css file

Reply:
Thanks, changed it